src/uct/ib/dc/dc_mlx5.c

```diff
@@ -534,6 +541,12 @@ uct_dc_mlx5_iface_create_dct(uct_dc_mlx5_iface_t *iface,
          goto err;
     }
 
+    if (uct_ib_set_ece(dev, iface->rx.dct.verbs.qp,
+                       iface->super.super.config.ece) != UCS_OK) {
+        ucs_error("error setting qp ece");
```

Comment:
1. move this error print inside uct_ib_set_ece
2. use status variable

Reply:
Done